### DISCUSSION TITLE
How to properly setup ESLint v9 with Next and Mantine configs?

### DISCUSSION THREAD

**Discussion by joepetrillo** [mantinedev/mantine#7251] [Q&A]
Next now supports ESLint v9, but I am not sure how to use the `eslint-config-next` alongside `eslint-config-mantine`, any ideas?

Currently have this in my config file, but get a NextJS warning every time I run the lint command.
"The Next.js plugin was not detected in your ESLint configuration."

```js
import mantine from "eslint-config-mantine";
import tseslint from "typescript-eslint";

export default tseslint.config(...mantine, {
  ignores: ["**/*.{mjs,cjs,js,d.ts,d.mts}"],
});
```
![image](https://github.com/user-attachments/assets/ffd68c25-71cf-4a62-a0f1-c6ac2cc42a6d)

**Comment by MikeSuiter** [comment]
@joepetrillo did you even come up with a solution to get both Mantine and nextjs eslint in a project?
